language-design.md

```diff
@@ -48,8 +119,13 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 In what way is an API a language? 
 
 **Response**
-
-
+The notion of "when in doubt, leave it out" is very linked to DSLs and languages in general 
+[Bloch, 2006]. In terms of DSLs, we discussed in class how DSLs should support the 
+bare minimum required to run the programs they want to run. Otherwise, they risk 
+becoming a general purpose language and losing the specificity of a DSL. Additionally, this 
+is true of languages in general. In English, students are taught to be concise and focus on one topic. 
+Similarly, the "when in doubt, leave it out" mentality means that APIs need to leave out 
+anything that takes focus away from its main purpose.
```

Comment:
Conciseness is a nice property for APIs to have, but I'm still wondering in what ways an API is a language? I think I was expecting properties of APIs that satisfy our definition and intuition of a programming language.